### PR TITLE
updating twitter keyword vars

### PR DESCRIPTION
**Pull request by fivetran-sheringuyen**
**Are you a current Fivetran customer?** 

Fivetran Employee

**What change(s) does this PR introduce?** 

Updating `twitter_ads__using_keywords` variables for default consistency 

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature [#69]
- [ ] No 

**How did you test the PR changes?** 

 

- [x]Buildkite  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

**Comment by fivetran-sheringuyen** [comment]
- [ ] Regenerate docs
- [ ] Update Changelog
- [ ] Update dbt projects (find out whether this is a patch or minor)

File: integration_tests/requirements.txt [hunk 1]
```diff
@@ -3,4 +3,5 @@ dbt-bigquery>=1.0.0
 dbt-redshift>=1.0.0
 dbt-postgres>=1.0.0
 dbt-spark>=1.0.0
-dbt-spark[PyHive]>=1.0.0
\ No newline at end of file
+dbt-spark[PyHive]>=1.0.0
```
Comment (fivetran-avinash) [inline integration_tests/requirements.txt:6]: Shouldn't requirements also have a min of <1.3.0 for each DB type like the other packages?

File: packages.yml [hunk 2]
```diff
@@ -8,8 +8,11 @@ packages:
   - package: fivetran/facebook_ads
     version: [">=0.5.0", "<0.6.0"]
 
-  - package: fivetran/google_ads
-    version: [">=0.8.0", "<0.9.0"]
+  # - package: fivetran/google_ads
+  #   version: [">=0.8.0", "<0.9.0"]
+  - git: https://github.com/fivetran/dbt_google_ads.git
```
Comment (fivetran-avinash) [inline packages.yml:13]: This should be reverted back to the original commented out `google_ads` package before merging I think?

**Review by fivetran-avinash — COMMENTED** [review]
@fivetran-sheringuyen Looks mostly good! Only a few suggested changes.

File: .buildkite/scripts/run_models.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euo pipefail
```
[the remark this answers is not in the document]
Comment (fivetran-sheringuyen) [reply .buildkite/scripts/run_models.sh:3]: same responses!

File: integration_tests/requirements.txt (hunk 1 above)
Comment (fivetran-sheringuyen) [reply integration_tests/requirements.txt:6]: good catch! updated

File: packages.yml (hunk 2 above)
Comment (fivetran-sheringuyen) [reply packages.yml:13]: yep!

**Review by fivetran-avinash — APPROVED** [review]
@fivetran-sheringuyen Approved!

**Pull request merged**